include/JS/login.js

```diff
@@ -0,0 +1,72 @@
+
+// ********************************************************************************************************************************
+//when the user changes something in the login form 
+// *********************************************************************************************************************************
+document.addEventListener('input', function (event) {
+
+    // to hide the error message once the user starts typing
+    if (event.target.id === 'userEmail'){
+
+        if(document.getElementById('emailErr').style.visibility == 'visible'){
+            document.getElementById('emailErr').style.visibility = 'hidden';          
+        }        
+    }
+
+    if (event.target.id === 'userPassw'){
+              
+        if(document.getElementById('passwErr').style.visibility == 'visible'){
+            document.getElementById('passwErr').style.visibility = 'hidden';           
+        }  
+    }
+});
+
+// ********************************************************************************************************************************
+//when the user submit the the login form 
+// *********************************************************************************************************************************
+//see if I can do AJAX request from here?
+
+document.getElementById('loginForm').addEventListener('submit', function (event) {
+
+    let input = {       
+        Email: document.getElementById('userEmail').value,
+        Password: document.getElementById('userPassw').value
+    };
+
+    let admin = new FormData();
+    admin.append( "CheckAdmin", JSON.stringify(input)); 
```

Comment:
In theory, you shouldn't need to stringify the data when you're building the request. AJAX libraries usually handle that step for you, and then PHP will automatically JSON decode it so you can access it like you normally do